### PR TITLE
doc(whl): polish doc for data_helper, model_helper, parameter, metric, math_helper, backend_helper

### PR DESCRIPTION
**Pull request by kxzxvbk**
## Check List

- [ ] merge the latest version source branch/repo, and resolve all the conflicts
- [ ] pass style check
- [ ] pass all the tests

### CODE REVIEW

File: ding/torch_utils/data_helper.py [hunk 1]
```diff
@@ -16,13 +16,18 @@
...
+        - device (:obj:`str`): The device wanted.
+        - ignore_keys (:obj:`list`): The keys to be ignored in transfer, default set to empty.
     Returns:
-        - item (:obj:`Any`): the transferred item
+        - item (:obj:`Any`): The transferred item.
+    Examples:
+        >>> setup_data_dict['module'] = nn.Linear(3, 5)
+        >>> device = 'cuda'
+        >>> cuda_d = to_device(setup_data_dict, device, ignore_keys=['module'])
+        >>> assert cuda_d['module'].weight.device == torch.device('cpu')
```
Comment (PaParaZz1) [inline ding/torch_utils/data_helper.py:30]: add another key-value which is not ignored

**Review by PaParaZz1 — APPROVED** [review]
